content/en/docs/install-operate/installation.md

```diff
@@ -307,28 +307,28 @@ In these steps, we are targeting a Debian-like system on `x86_64` architecture.
     ```
 
 4. Run `falco-driver-loader` binary to install the kernel module or the BPF probe. If you want to use other sources like the modern BPF probe or plugins you can skip this step.
-> To install the driver, the `falco-driver-loader` script requires write and execution permissions on the `/tmp` directory, as it will try to create and execute a script from there.
+   > To install the driver, the `falco-driver-loader` script requires write and execution permissions on the `/tmp` directory, as it will try to create and execute a script from there.
```

Comment:
```suggestion
   {{% pageinfo color="info" %}}

   To install the driver, the `falco-driver-loader` script requires write and execution permissions on the `/tmp` directory, as it will try to create and execute a script from there.
   
   {{% /pageinfo%}}
```